src/ConfigurationProvider.php

```diff
@@ -60,9 +60,7 @@ public function register(Container $app)
                 return ConfigurationFactory::init($app['config.driver'], ConfigurationFactory::ENV_COMMON, [], null)
                     ->getSettings();
             } catch (Exception $e) {
-                unset($e);
-                //It will return exception if common file not exists. We don't bother of that : ).
-            } finally {
+                // This means that the common file doesn't exist. Not a problem.
                 return [];
```

Comment:
Actually, when you do a `return` in normal flow and a `return` inside `finally`, the one inside `finally` will _overwrite_ the original return. Weird, eh?